pkg/master/master.go

```diff
@@ -117,6 +117,30 @@ func (s *server) Package(stream service.BuildService_PackageServer) error {
 	return stream.SendAndClose(&service.BuildResponse{Success: true})
 }
 
+func (s *server) Project(stream service.BuildService_ProjectServer) error {
```

Comment:
Project is a noun and does not reflect what the function does. So please choose a proper name.